### ISSUE TITLE
preventPageScrolling on touch devices

### ISSUE THREAD

**Issue by erikhagreis** [jamesflorentino/nanoScrollerJS#295]
It seems there is no support for preventPageScrolling on touch devices. I have looked into the code, and cannot find logic in the `preventScrolling` function that handles events of type `touchmove`. There is only logic for `DOMMouseScroll' and 'wheel' events.

Is this on purpose, or am I missing anything? It would be feasible for me to create a pull request where also touch events are handled in this feature.

**Comment by kristerkari** [comment]
Yeah it is true because I guess nanoScroller was initially created with desktop browsers in mind.

If `preventScrolling` can be implemented so that it works consistently in mobile browsers, then there should be reason to support it as well.

**Comment by erikhagreis** [comment]
I'm currently working on an implementation and learning a lot about how touchdevices handle scrolling. Once I understand enough of it I'll come back to this.

**Comment by kristerkari** [comment]
@erikhagreis sounds good! You should share your learnings when you are done :) :+1:
